src/types/library/library_item.rs

```diff
@@ -40,11 +46,22 @@ impl LibraryItem {
     pub fn should_sync<E: Env + 'static>(&self) -> bool {
         let year_ago = E::now() - Duration::days(365);
         let recently_removed = self.removed && self.mtime > year_ago;
-        self.r#type != "other" && (!self.removed || recently_removed)
+        self.r#type != MetaType::Other && (!self.removed || recently_removed)
     }
     #[inline]
     pub fn is_in_continue_watching(&self) -> bool {
-        self.r#type != "other" && (!self.removed || self.temp) && self.state.time_offset > 0
+        self.r#type != MetaType::Other && (!self.removed || self.temp) && self.state.time_offset > 0
```

Comment:
We should not include the LibraryItem in CW if we've watched a certain amount of the Movie/series.

@jaruba mentioned that this is working very well in the Android TV App so we should look into adding the same logic here as well.